src/doc/rustc/src/command-line-arguments/print-options.md

```diff
@@ -0,0 +1,212 @@
+# Print Options
+
+All of these options are passed to `rustc` via the `--print` flag.
+
+Those options prints out various information about the compiler. Multiple options can be
+specified, and the information is printed in the order the options are specified.
```

Comment:
Discussion: should be explicitly point out something like, unless the output format is explicitly documented as a stability guarantee, then the output format may be intended for human consumptions only and may change between compiler versions?

(Meta: this is of course in comparison to something that *would* be first-class designed to be machine-consumable, like a `--print-json=...` e.g. [#t-compiler/major changes > Add &#96;--print=supported-crate-types&#96; compiler-team#836 @ 💬](https://rust-lang.zulipchat.com/#narrow/channel/233931-t-compiler.2Fmajor-changes/topic/Add.20.60--print.3Dsupported-crate-types.60.20compiler-team.23836/near/499736720))

Reply:
EDIT: opened zulip thread https://rust-lang.zulipchat.com/#narrow/channel/131828-t-compiler/topic/Print.20stability.20guarantees.20and.20the.20lack.20thereof/with/507641177